workflow/hosts/hercules.yaml

```
@@ -9,7 +9,7 @@ STMP: '/work/noaa/stmp/${USER}/HERCULES'
 PTMP: '/work/noaa/stmp/${USER}/HERCULES'
 NOSCRUB: $HOMEDIR
 SCHEDULER: slurm
-ACCOUNT: fv3-cpu
+ACCOUNT: da-cpu
```

I would prefer this to stay `fv3-cpu`.  Now that #3127 has been merged, the account should be specifiable by setting `HPC_ACCOUNT` and running `create_experiment.py` or passing the `-A <account>` option to either `generate_workflows.sh` or `setup_expt.py`.

```suggestion
ACCOUNT: fv3-cpu
```